adapters/silvermob/params_test.go

```diff
@@ -0,0 +1,55 @@
+package silvermob
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// TestValidParams makes sure that the silvermob schema accepts all imp.ext fields which we intend to support.
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderSilverMob, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected silvermob params: %s", validParam)
+		}
+	}
+}
+
+// TestInvalidParams makes sure that the silvermob schema rejects all the imp.ext fields we don't support.
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, invalidParam := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderSilverMob, json.RawMessage(invalidParam)); err == nil {
+			t.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"zoneid": "16", "host": "us"}`,
+	`{"zoneid": "16", "host": "eu"}`,
+}
+
+var invalidParams = []string{
+	``,
+	`null`,
+	`true`,
+	`5`,
+	`4.2`,
+	`[]`,
+	`{}`,
+	`{"ZoneID": "asd", "Host": "123"}`,
+	`{}`,
+	`{"ZoneID": "asd"}`,
+	`{"Host": "111"}`,
```

Comment:
You may want to add examples where the properties are correct ("zoneid" and "host"), but the values are not strings.

Reply:
Added `{"zoneid": 16, "host": 111}`,